src/list.tsx

```diff
@@ -37,6 +37,7 @@ interface CommonProps {
     children?: void; // no children allowed
     headline?: string | React.ReactNode;
     title: string;
+    titleAs?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
```

Comment:
Should always be a heading? I'm thinking for example in:

https://tinyurl.com/24aw45hx

Reply:
hmmm... good point. I guess it make sense to allow anything in rows. Still, do you think h3 is a good default or should we default to span?

Reply:
My first thought was to keep span as default but i don't have a stronger opinion rather than the sake of continuity.

Reply:
ok, I think it makes sense. Changed the default to span